Makefile

```diff
@@ -21,37 +21,23 @@ SHELL := /bin/bash
 # Current version, updated after a release.
 VERSION ?= "0.5.0"
 
-# If GOPATH is not set by the env, set it to a sane value
-GOPATH ?= $(shell cd ../../..; pwd)
-export GOPATH
-
-# If GOPATH is made up of several paths, use the first one for our targets in this Makefile
-GO_TOP := $(shell echo ${GOPATH} | cut -d ':' -f1)
-
-export CGO_ENABLE=0
-
 # OUT is the directory where dist artifacts and temp files will be created.
-OUT=${GO_TOP}/out
+OUT := out
 
-GO ?= go
+# Environment for tests, the directory containing istio and deps binaries.
+# Typically same as GOPATH/bin, so tests work seemlessly with IDEs.
+ISTIO_BIN := bin
```

Comment:
should probably be `build/` or `output/` or such but not the same directory as an existing directory from the source tree

Reply:
it's difficult to find out all ways "bin" used throughout except by trial and error. using CI as my  linux machine.

Reply:
that's another good reason to use something else than bin to find sooner than later the hardcoded ones

Reply:
agree, but that's besides the mac build goal of this PR.

Reply:
Don't know about mac build goals - but this is affecting non-mac builds, which is not good./